lib/plugins/aws/provider.js

```diff
@@ -285,6 +286,10 @@ class AwsProvider {
             type: 'string',
             pattern: '^arn:',
           },
+          awsArnList: {
+            type: 'array',
+            items: { $ref: '#/definitions/awsArnString' },
+          },
```

Comment:
Let's not introduce such definition. I think we may find tens of other cases where we could go in such direction, while it quickly pollutes the definitions with noise. 

Also seeing right at the point that we deal with an _array_ and not a singular value, has a value.